src/linear_algebra/basic.lean

```diff
@@ -571,6 +570,19 @@ end comm_ring
 
 end linear_map
 
+/--
+The `ℤ`-linear equivalence between additive morphisms `A →+ B` and `ℤ`-linear morphisms `A →ₗ[ℤ] B`.
+-/
+@[simps]
+def add_monoid_hom_lequiv_linear_map {A B : Type*} [add_comm_group A] [add_comm_group B] :
```

Comment:
Could you please duplicate this for `nat`? I guess we should tag a `_nat` or `_int` to the end of the name?
Should the name be shortened to `add_monoid_hom_lequiv_nat` etc?

Reply:
Can we duplicate it for `rat` / `division_ring` too?

Reply:
Okay, I've done the `nat` version. I'm not planning on thinking about a `rat` version for now.